### PR TITLE
Fixed problem with birthday and increased max length for field "Навчальний заклад"

### PR DESCRIPTION
**Pull request by petromonka**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/userPage/EditUserPage/EditUserPage.tsx [hunk 1]
```diff
@@ -417,6 +417,11 @@ export default function () {
     setPhotoName(defaultPhotoName);
   };
 
+  const handleDate = (datestr : string):Date =>{
+    var date = new Date(datestr);
+    return new Date( date.getTime() + Math.abs(date.getTimezoneOffset()*60000) )
```
Comment (OleksandrBilan) [inline src/pages/userPage/EditUserPage/EditUserPage.tsx:422]: magic number 60000. Create a variable for this

**Review by juliazhorniakAMI — APPROVED** [review]

File: src/pages/userPage/EditUserPage/EditUserPage.tsx [hunk 2]
```diff
@@ -181,7 +181,7 @@ export default function () {
       { pattern: allVariantsPattern, message: wrongAllVariantsMessage },
     ],
     placeOfStudy: [
-      { max: 50, message: maxLength(50) },
+      { max: 100, message: maxLength(100) },
```
Comment (christykobyletska) [inline src/pages/userPage/EditUserPage/EditUserPage.tsx:184]: Have you changed maxLength of place of study on the backend?

**Review by christykobyletska — CHANGES_REQUESTED** [review]

File: src/pages/userPage/EditUserPage/EditUserPage.tsx (hunk 2 above)
Comment (petromonka) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx:184]: Yes I changed maxLength on backend too and will make pull request today ;)
```suggestion
      { max: 100, message: maxLength(100) },
```

Comment (christykobyletska) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx:184]: ok

**Review by christykobyletska — APPROVED** [review]

**Review by VolodymyrKostiv — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Pull request closed without merging**